docs/home/contribute/generated-reference/federation-components.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Generating Reference Pages for Kubernetes Federation Components
```

Comment:
This page is not linked from docs-home.html ?

Reply:
There's a new home page coming soon as part of the User Journeys project. We'll revisit linking when that is in place.